typings.json

```diff
@@ -1,9 +1,5 @@
 {
   "name": "sequelize",
   "main": "index.d.ts",
-  "version": "4.0.0-1",
-  "dependencies": {
-    "bluebird": "registry:npm/bluebird#3.5.0+20170314181206",
-    "validator": "registry:npm/validator#5.7.0+20160912014851"
-  }
+  "version": "4.0.0-1"
```

Comment:
This is a breaking change for `typings` users.

Reply:
I kept getting nasty warnings about `typings` being deprecated, and this should work regardless as the typings will be installed automatically via npm's `@types` instead, I testes this in 2 projects and couldn't find any type issues.

Reply:
In reference to the `validator` problem, why not just drop support for typings, as mentioned they are deprecated.

Reply:
I will rollback these changes for now, but don't expect them to work, lol.

Reply:
Just because they are deprecated doesn't mean there aren't still people using it.
@blakeembrey do you have stats on this?

Reply:
@felixfbecker I do have analytics, but I'd be happy to start removing everything `typings` related. It won't break backward compat thankfully since I was super persistent with hard tagging everything to a specific commit. The only people it'll break for it already warned them about it, and they can always just go back to an old commit until they move to `@types` (but I'm pretty confident this won't be an issue for anyone).

Reply:
Okay, lets remove it then